script.skin.helper.service/plugin.py

```diff
@@ -46,8 +46,36 @@ def __init__(self):
             elif action == "LAUNCH":
                 xbmcplugin.setResolvedUrl(handle=int(sys.argv[1]), succeeded=False, listitem=xbmcgui.ListItem())
                 path = sys.argv[2].split("&path=")[1]
-                xbmc.executebuiltin("Action(Close)")
                 xbmc.executebuiltin(path)
+            elif action == "FOCUSANDCLICK":
+                #used as workaround to display local media in extendedinfo actorinfo
```

Comment:
You're the best ;-) Also great PR with the ListItem.DBID, I'm indeed also working around that with the ListItem.property(dbid) in skinhelper.

In the meanwhile I've added another commit (didn't squash it) to remove the ugly hack and fix the few typos you pointed me to. I will adjust my skin to check against ListItem.Property(dbid) in the extendedinfo actor dialog to show some overlay to notify it's in the library.

Thanks!